core/tests/protractor_utils/workflow.js

```diff
@@ -174,6 +174,7 @@ var createAndPublishExploration = async function(
   if (language) {
     await explorationEditorSettingsTab.setLanguage(language);
   }
+  await explorationEditorSettingsTab.enableCorrectnessFeedback();
```

Comment:
Why this change? Can you please explain?

Reply:
I added validation to the chapter addition modal, so that it prevents explorations from being added if their correctness feedback is disabled. In the e2e test which tests chapter addition functionality, it performs this test by trying to add explorations whose correctness feedback is not enabled. But the exploration isn't accepted and the test fails because of the checks I added.

Hence I added a line to enable correctness feedback of the explorations when this function is called in the test.

Reply:
Thanks for the explanation.